dashboard/test/utils/golden.dart

```diff
@@ -50,8 +50,14 @@ class CocoonFileComparator extends LocalFileComparator {
     return super.getGoldenBytes(golden);
   }
 
-  static bool get _isLuciCi {
-    return !kIsWeb && Platform.environment.containsKey('LUCI_CONTEXT');
+  static bool get _isRunningOnCi {
```

Comment:
should we log that we're running on CI so.. you know... we see it?

actions are setup to run on ubuntu, so it should be the same locally or on github as well.